cmd/goal/network.go

```diff
@@ -246,3 +279,64 @@ var networkDeleteCmd = &cobra.Command{
 		reportInfof(infoNetworkDeleted, networkRootDir)
 	},
 }
+
+var networkGenesisCmd = &cobra.Command{
+	Use:   "genesis",
+	Short: "Creates the genesis.json, root and participation keys for a wallet",
+	Args:  validateNoPosArgsFn,
+	Run: func(cmd *cobra.Command, _ []string) {
+		var err error
+		if networkRootDir != "" {
+			reportErrorf("This command does not take a network directory as an argument. Use --genesisdir flag instead.")
+		}
+
+		genesisDir, err = filepath.Abs(genesisDir)
+		if err != nil {
+			panic(err)
```

Comment:
nit: maybe use reportErrorf instead of panic?

Reply:
We generally seem to panic for filepath errors: https://github.com/algorand/go-algorand/blob/master/cmd/goal/network.go#L89

I'm fine either way, but just followed existing conventions here.